modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.2.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:EModelBuildingShape",
+      "@type": "sh:NodeShape",
+      "label": "Emodel building shape",
+      "comment": "Get all optimizations for PC CA1",
+      "targetClass": "nsg:EModelBuilding",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.1/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:version",
+              "name": "Version",
+              "description": "Version of the generated emodel.",
+              "datatype": "xsd:string",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:bestScore",
```

Comment:
Why the bestScore is removed from the emodelgeneration ?

Reply:
We have a custom activity, so why not put all the needed properties on this activity(bestScore was moved) instead of having generation objects?